### PR TITLE
Added example for the DS18B20

### PR DESCRIPTION
**Pull request by pedrominatel**
# ESP-BSP Pull Request checklist

> Note: For new BSPs create a PR with this [link](https://github.com/espressif/esp-bsp/compare/main...my-branch?quick_pull=1&template=pr_template_bsp.md).
- [ ] Version of modified component bumped
- [x] CI passing

# Change description

This PR adds a simple example of how to read the DS18B20. This example was used during a presentation to show how to use the components and BSP.

### CODE REVIEW

File: components/ds18b20/examples/ds18b20-read/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# DS18B20 sensor example
+
+This example shows how to use the DS18B20 component.
+
+## How to use the example
+
+### Hardware Required
+
+* An ESP development board
```
[the remark this answers is not in the document]
Comment (pedrominatel) [reply components/ds18b20/examples/ds18b20-read/README.md:9]: Thank you @igrr for your review. I'll do it.

**Review by igrr — COMMENTED** [review]
LGTM!

To fix the build issue, I think you need to add a rule in .build-test-rules.yml to only build the example if the chip has RMT:

```yml
components/ds18b20:
  disable:
    - if: SOC_RMT_SUPPORTED != 1
      reason: Onewire component depends on RMT peripheral
```

and please clean up commit history before merging.

**Comment by pedrominatel** [comment]
> LGTM!
> 
> To fix the build issue, I think you need to add a rule in .build-test-rules.yml to only build the example if the chip has RMT:
> 
> ```yaml
> components/ds18b20:
>   disable:
>     - if: SOC_RMT_SUPPORTED != 1
>       reason: Onewire component depends on RMT peripheral
> ```
> 
> and please clean up commit history before merging.

Can you kindly take a look, @tore-espressif? Many thanks!

**Review by tore-espressif — APPROVED** [review]
LGTM!

**Pull request merged**